charts/weave-gitops/templates/wego-app.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: wego-app
+  namespace: {{.Values.namespace}}
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: wego-app
+    spec:
+      serviceAccountName: wego-app-service-account
+      containers:
+        - name: wego-app
+          image: "ghcr.io/weaveworks/wego-app:{{.Chart.AppVersion}}"
+          args: ["ui", "run", "-l", "--helm-repo-namespace","{{.Values.namespace}}"]
+          ports:
+            - containerPort: 9001
+              protocol: TCP
+          imagePullPolicy: IfNotPresent
+  selector:
+    matchLabels:
+      app: wego-app
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: read-resources
+subjects:
+  - kind: ServiceAccount
+    name: wego-app-service-account
+    namespace: {{.Values.namespace}}
+roleRef:
+  kind: ClusterRole
+  name: resources-reader
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: resources-reader
```

Comment:
The name of this doesn't really make sense.  It should be something along the lines of weave-gitops-role.  This will be removed in a future PR.